terraform/outputs.tf

```diff
@@ -0,0 +1,52 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+output "app_name" {
+  description = "Name of the deployed application."
+  value       = juju_application.grafana-agent-k8s.name
+}
+
```

Comment:
Just a suggestion, but I've been adding comments to separate required integration from provided integrations. I think this improves readability.